### ISSUE TITLE
Mark Lines have Bad Anchors

### ISSUE THREAD

**Issue by johannes-wolf** [johannes-wolf/cetz#247]
See: #246. The anchors seem to get moved with the mark, which is wrong.

**Comment by johannes-wolf** [comment]
Can you post the source used to generate the image? To get the start/end of a line use the `"start"` and `"end"` anchors, as `"left"`, `"right"`, ... default to the bounding box, which does not take the stroke into account and therefore is a bit smaller than expected!

**Comment by JuliDi** [comment]
Sure, this is where it happens:
```typst
#import "@preview/cetz:0.1.2": canvas, draw, tree

#canvas(length: 1cm, {
  import draw: *

  rect((0,0), (1,1), name: "rect")
  line("rect.top", (rel: (0, 2)), mark: (start: ">"), stroke: 2pt)
})
```
Seems to be caused directly by the stroke. With default stroke it looks alright.

**Comment by johannes-wolf** [comment]
Fixed with #314.
